crates/air_r_formatter/src/r/auxiliary/call_arguments_table.rs

```diff
@@ -0,0 +1,508 @@
+    // at the same time, if there is an argument like `1.`
+    has_decimal: bool,
+    max_width: usize,
+    max_value_width: usize,
+    max_integer_part: usize,
+    max_fractional_part: usize,
+}
+
+impl FormatRCallArguments {
+    pub(crate) fn fmt_table(
+        &self,
+        node: &RCallArguments,
+        f: &mut RFormatter,
+    ) -> FormatResult<Option<()>> {
+        let l_token = node.l_paren_token()?;
+        let r_token = node.r_paren_token()?;
+        let args = node.items();
+
+        if args.is_empty() {
+            return self.fmt_call_like(node, f).map(Some);
+        }
+
+        // Get table and alignment info
+        let table = match build_table(&args, f)? {
+            Some(table) => table,
+            None => return Ok(None),
+        };
+        let rows = table.rows;
+        let column_info = table.cols;
+
+        // Format with alignment
+        let formatted_table = format_with(|f| {
+            for (row_i, row) in rows.iter().enumerate() {
+                if row_i > 0 {
+                    // Rows are separated with hard line breaks because the
+                    // lines of arguments in table calls should never be
+                    // rearranged by the formatter.
+                    write!(f, [hard_line_break()])?;
+                }
+
+                for (col_j, arg_data) in row.iter().enumerate() {
+                    let (left_pad, right_pad) = if column_info[col_j].max_width > 0 {
+                        column_info[col_j].padding(&arg_data.kind)
+                    } else {
+                        // Empty columns are not padded, so that commas stick to
+                        // each other as in regular calls: `list(,,,)`
+                        (0, 0)
+                    };
+
+                    // Add left padding for right-aligned values
+                    write_spaces(left_pad, f)?;
+
+                    // Format the value
+                    match &arg_data.kind {
+                        ArgKind::Other { text } => {
+                            let arg_syntax = arg_data.node.syntax();
+
+                            // We've formatted the argument without comments, so
+                            // we're in charge of formatting them
+                            format_leading_comments(arg_syntax).fmt(f)?;
+                            disable_skip_comments(arg_syntax, f);
+
+                            // 0-length arguments are holes. Don't print them
+                            // because a `text("")` after a `Space` will prevent
+                            // the latter from being considered trailing by the
+                            // printer, and won't be removed if trailing.
+                            if !text.is_empty() {
+                                write!(f, [dynamic_text(text, 0.into())])?;
+                            }
+
+                            format_trailing_comments(arg_data.node.syntax()).fmt(f)?;
```

Comment:
```suggestion
                            let arg_syntax = arg_data.node.syntax();

                            // Suppression comments do nothing inside a table
                            f.comments().mark_suppression_checked(arg_syntax);

                            // We've formatted the argument without comments, so
                            // we're in charge of formatting them
                            format_leading_comments(arg_syntax).fmt(f)?;

                            // 0-length arguments are holes. Don't print them
                            // because a `text("")` after a `Space` will prevent
                            // the latter from being considered trailing by the
                            // printer, and won't be removed if trailing.
                            if !text.is_empty() {
                                write!(f, [dynamic_text(text, TextSize::from(0))])?;
                            }

                            format_trailing_comments(arg_syntax).fmt(f)?;
```

- Inline `disable_skip_comments()` with appropriate comment
- Consistently use `arg_syntax`

Reply:
yep nice